### ISSUE TITLE
Rust main track blocked

### ISSUE THREAD

**Issue by Caspinol** [exercism/exercism#4319]
Hello,
I recently came back to exercism after a break. I noticed that all my progress was succesfully migrated from the old version to the new version however after completing few more exercises no mentor have approved them so far after almost 3 days.

Is this a issue of not enough mentors or is there a problem with my profile after the migration?

Regards,
K

**Comment by iHiD** [comment]
Did you close this because it solved itself?

**Comment by Caspinol** [comment]
Yes, it turned out to be mentors swamped with too many students.
